src/plugins/discover/public/plugin.tsx

```diff
@@ -211,10 +211,7 @@ export class DiscoverPlugin
   private urlGenerator?: DiscoverStart['urlGenerator'];
   private locator?: DiscoverAppLocator;
 
-  setup(
```

Comment:
one note about moving to an async setup
/**
 * A plugin with asynchronous lifecycle methods.
 *
 * @deprecated Asynchronous lifecycles are deprecated, and should be migrated to sync {@link Plugin | plugin}
 * @public
 */

Reply:
Removed `async` setup.